DIPs/accepted/DIP1018.md

```diff
@@ -10,24 +10,7 @@
 
 ## Abstract
 
-This document proposes copy constructor semantics as an alternative
-to the design flaws and inherent limitations of the postblit.
-
-### Reference
```

Comment:
Is the removal of the reference section really necessary?

Reply:
It's not removed -- it's moved down to the bottom. I've modified the DIP template in the same way.

Reply:
Oh, whoop, didn't notice. Thanks!